components/gfx/platform/freetype/font.rs

```diff
@@ -138,20 +130,21 @@ impl PlatformFontMethods for PlatformFont {
         _font_identifier: FontIdentifier,
         data: Arc<Vec<u8>>,
         face_index: u32,
-        pt_size: Option<Au>,
+        requested_size: Option<Au>,
     ) -> Result<PlatformFont, &'static str> {
-        let face = create_face(data.clone(), face_index, pt_size)?;
-        let mut handle = PlatformFont {
+        let face = create_face(data.clone(), face_index)?;
+        let requested_face_size = requested_size.unwrap_or_else(|| Au::from_f64_px(12.));
```

Comment:
Is the default value of `12.0` based on a CSS specification?

Reply:
Nope...this should only be `None` when instantiating a `PlatformFont` without a size -- which happens only when creating a descriptor for system fonts. I've modified this though to only set the size when the requested size isn't `None` and giving a 0 size to the font itself in this case. This should make more obvious what it is going on.